bayem/vba.py

```diff
@@ -592,7 +594,7 @@ def summary(
             dist = p.dist()
 
             if gamma_as_sd:
-                entry = [name, "?", 1 / dist.mean() ** 0.5, "?"]
+                entry = [name, 1/dist.ppf(0.5)**0.5, 1 / dist.mean() ** 0.5, np.nan]
```

Comment:
Could we use this instead?
~~~py
[name, dist.median()**(-2), dist.mean()**(-2), dist.std()**(-2)]
~~~